.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-16.04 # https://launchpad.net/~hvr/+archive/ubuntu/ghc?field.series_filter=xenial
+          - ubuntu-18.04 # latest # https://launchpad.net/~hvr/+archive/ubuntu/ghc?field.series_filter=xenial
```

Comment:
@parsonsmatt not a big thing, but I guess we should remove the `latest` from the comment. Other than that it's good to go.